Gemfile

```diff
@@ -139,8 +139,8 @@ source "https://rails-assets.org" do
   gem "rails-assets-jquery-ujs",                      "~>1.1.0"
   gem "rails-assets-jqueryui",                        "~>1.9.2"
   gem "rails-assets-bootstrap-switch",                "~>3.3.2"
-  gem "rails-assets-angular-bootstrap-switch",        "~>0.4.1"
-  gem "rails-assets-kubernetes-topology-graph",       "= 0.0.22"
+  gem "rails-assets-romanblanco--angular-bootstrap-switch",        "~>0.4.2"
+  gem "rails-assets-romanblanco--topology-graph",       "= 0.0.23"
```

Comment:
@abonas Yes, once both PRs _(https://github.com/kubernetes-ui/topology-graph/pull/18 and 
https://github.com/frapontillo/angular-bootstrap-switch/pull/103)_ are merged, I'll take back this change. I'm using it in the meantime to be able to continue with my work.